metadata-ingestion/src/datahub/ingestion/source/aws/sagemaker_processors/common.py

```diff
@@ -1,13 +1,22 @@
-from dataclasses import dataclass
-from typing import Dict, Optional, Union
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional, Union
 
 from pydantic.fields import Field
 
-from datahub.ingestion.api.source import SourceReport
+from datahub.configuration.source_common import PlatformInstanceConfigMixin
 from datahub.ingestion.source.aws.aws_common import AwsSourceConfig
+from datahub.ingestion.source.state.stale_entity_removal_handler import (
+    StaleEntityRemovalSourceReport,
+    StatefulIngestionConfigBase,
+    StatefulStaleMetadataRemovalConfig,
+)
 
 
-class SagemakerSourceConfig(AwsSourceConfig):
+class SagemakerSourceConfig(
+    AwsSourceConfig,
+    PlatformInstanceConfigMixin,
```

Comment:
it doesn't look like this source supports platform instances, so having it in the config might be misleading

Reply:
Thank you! It's interesting that if I give `platform_instance` in the ingestion recipe, then I can see the platform instance available when browsing ML models, it's in the response of graphQL query `getBrowseResultsV2` even though each ML models entity does not have `platform_instance`.

Do you think I can make some more improvements to support platform instances? Most of the `make_ml_model..*_urn` code is in `mce_builder.py`.